docs/tendermint-core/README.md

```diff
@@ -18,3 +20,4 @@ This section dives into the internals of Go-Tendermint.
 - [State Sync](./state-sync.md)
 - [Mempool](./mempool.md)
 - [Light Client](./light-client.md)
+- [Reactors](./reactors/README.md)
```

Comment:
Do you think we should have a reactors section if we already have a `Mempool`, `StateSync`, `FastSync` sections. I would say we just merge them with the existing ones if the are there.

And then we just need to add `Consensus`, `Evidence` and `PeerExchange`

Reply:
If we make all of these private/internal, maybe we want to think about how we document them?

Reply:
The docs here are meant more as a implementation spec, this is orthogonal to internal.